frontend/src/components/data-table/charts/components/form-fields.tsx

```diff
@@ -670,32 +671,64 @@ export const TooltipSelect = ({
   saveFunction: () => void;
 }) => {
   const form = useFormContext();
+  const formValues = useWatch({ control: form.control });
   const { fields } = useChartFormContext();
+
+  // TODO: This hack allows users to deselect, but in return,
+  // whenever component is re-toggled, this gets reset
+  const [touched, setTouched] = React.useState(false);
```

Comment:
another approach could be to have `"Default"` as an option. and whenever that get's selected, it's mutually exclusive from every other option.